KIPs/kip-xx.md

```diff
@@ -0,0 +1,52 @@
+---
+kip: to be assigned
+title: Transaction data gas cost reduction
+author: Lake (@hyunsooda) and Ollie (@blukat29)
+discussions-to: https://github.com/kaiachain/kips/issues/22
+status: Draft
+type: Standards Track
+category: Core
+created: 2024-10-18
+---
+
+## Simple Summary
+
+Changing to calldata gas cost.
+
+## Abstract
+
+This KIP introduces a new tag for calldata gas payment, defining a calculation method that distinguishes between zero and non-zero bytes, with distinct costs assigned to each byte.
+
+## Motivation
+
+TBD
+
+## Specification
+
+The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in RFC 2119.
```

Comment:
We're not using these keywords. So I guess it's unnecessary.

Reply:
Removed.